samples/dashboards/demo/live-weather/demo.css

```diff
@@ -440,10 +431,16 @@
  *
  * */
 
-div#description {
-    padding: 15px;
+.highcharts-description {
+    padding: 0 20px;
 }
 
+/* *
+ *
+ *  Responsive
+ *
+ * */
```

Comment:
1. If we add the "responsive" header it should be applied to all demos. Otherwise, it should be removed from here.
2. We use 

```
/**
*
**/
```
without extra spaces in the first and the last line.

Reply:
I removed it and fixed the comment begin/end in some other files.